app/components/Thread/index.jsx

```diff
@@ -141,7 +141,7 @@ const Thread = React.createClass({
                     email_subject: `[RhinoBird] ${LoginStore.getUser().realname} mentioned you in comment`,
                     email_body: `${LoginStore.getUser().realname} mentioned you in comment of <a href="${window.location.href.split('#')[0]}">${this.props.threadTitle}</a>
                     <br/><br/> ${comment}`,
-                    url: `window.location.pathname#${c.cid}`
+                    url: window.location.href.split('#')[0]+"#" + c.cid
```

Comment:
The previous line `window.location.pathname#${c.cid}` dosen't give the right url. The variable window.location.pathname isn't populated with a true value, but treated as a plain text string.